Documentation/ColumnsConfig/Type/Group.rst

```diff
@@ -119,6 +119,27 @@ Group file relation
 
    Group file relation with some selected files (group_file_1)
 
+.. code-block:: php
```

Comment:
We should drop the whole example instead:

> Types “file” and “file_reference” should not be used anymore. Use FAL references instead.

Also see https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.0/Breaking-87193-DeprecatedFunctionalityRemoved.html

Reply:
Ah, sorry about that. I created the issue and tagged it with "good first issue". Was not the best choice.

@runepiper Thank you for your pull request. We should do what @mbrodala wrote.
 
You can make additional changes to this pull request or someone else can finish it off (since this is your first PR and I am not sure how familiar you are with the workflow).

Just let us know if you wish to continue here.

Reply:
Just added a commit which removes the whole example.

Reply:
Thanks. :-)